kibble/api/bundles_test.go

```diff
@@ -65,7 +66,7 @@ func TestBundlesToSeoDefaultImage(t *testing.T) {
 
 	serviceConfig := models.ServiceConfig{
 		"default_image_type": "landscape",
-		"image_root_path": "https://s3-bla-bla",
+		"image_root_path":    "https://s3-bla-bla",
```

Comment:
Nit
```suggestion
		"image_root_path": "https://s3-bla-bla",
```

Reply:
The added whitespace is correct.

Sincerely,

The Linter

Reply:
Dear Linter: that's some bullshit.